### ISSUE TITLE
Post-login navigation

### ISSUE THREAD

**Issue by MoShizzle** [tillywork/tillywork#43]
The user should never be on an empty page. We should always redirect the user after login to their last opened list or the first list if none exists.

**Comment by sagarhedaoo** [comment]
Hi! Can I start working on this?

**Comment by MoShizzle** [comment]
> Hi! Can I start working on this?

Absolutely, I assigned the issue to you. Feel free to ask here or on Discord for any help

**Comment by codonade** [comment]
Speaking of this, I think we should also allow users to choose a "startup list" which gets opened by default whenever they launch the application.

**Comment by Us3r-gitHub** [comment]
#124

**Comment by Us3r-gitHub** [comment]
It seems like the redirection to the first list isn't working. (Try open in incognito/private browser)
![image](https://github.com/user-attachments/assets/ecd00d16-0975-4314-bec6-3f4403d49bcf)

**Comment by codonade** [comment]
Yep. The PR got merged before implementing this, but it's a very trivial solution.
